zipkin-ui/js/component_data/default.js

```diff
@@ -3,11 +3,24 @@ import $ from 'jquery';
 import queryString from 'query-string';
 import {traceSummary, traceSummariesToMustache} from '../component_ui/traceSummary';
 
+export function parseApiQuery(windowLocationSearch) {
```

Comment:
made it convertToApiQuery. We can change this later, if you have a better name. I can't think of one.